src/overrides/gemini-scrollbar.less

```diff
@@ -1,4 +1,5 @@
-@import (inline) "../../node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
+// Make sure to import the following path before including this file
+// @import (inline) "node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
```

Comment:
is there any way to do it so that the developers would have to do it, rather than the end user? it's just more overhead for people wanting to use this module right? (after forcing them to install gemini in the first place)